common/callback.h

```diff
@@ -69,43 +68,45 @@ template<typename T> class GlobalFunctionCallback: public BaseCallback<T> {
 public:
 	GlobalFunctionCallback(GlobalFunction cb): _callback(cb) {}
 	virtual ~GlobalFunctionCallback() {}
-	virtual void operator()(T data) {
+	virtual void operator()(T data) { /*!< C function passed to the operator. */
 		if (_callback) _callback(data);
 	}
 };
 
 /**
  * Callback<T, S> implements an object-oriented callback.
  *
- * <T> stands for a class which method you want to call.
- * <S>, again, is the type of an object passed to operator().
+ * \<T\> stands for a class whose method you want to call.
+ * \<S\> is the type of the object passed to operator().
  *
  * So, if you have void MyClass::myMethod(AnotherClass) method,
  * the corresponding callback is Callback<MyClass, AnotherClass>.
- * You create it similarly to this:
+ * You can create it in the following way:
+ * @code
  *     new Callback<MyClass, AnotherClass>(
  *         pointerToMyClassObject,
  *         &MyClass::myMethod
  *     )
+ * @endcode
  */
 template<class T, typename S = void *> class Callback: public BaseCallback<S> {
 protected:
 	typedef void(T::*TMethod)(S);
 	T *_object;
-	TMethod _method;
+		;
```

Comment:
That broke the compilation horribly, for obvious reasons.

Reply:
To be honest, I was assuming we would still be working on this PR, at least adding missing doc in the high-priority headers. Hence the 'Do Not Merge' label in its title.